app/components/collections/information_component.rb

```diff
@@ -0,0 +1,41 @@
+# typed: false
+# frozen_string_literal: true
+
+module Collections
+  # Renders the information section of the collection (show page)
+  class InformationComponent < ApplicationComponent
+    sig { params(collection: Collection).void }
+    def initialize(collection:)
+      @collection = collection
+    end
+
+    sig { returns(Collection) }
+    attr_reader :collection
+
+    delegate :creator, :druid, :version,
+             to: :collection
+
+    sig { returns(String) }
+    def created
+      format_date_string(collection.created_at)
+    end
+
+    sig { returns(String) }
+    def last_saved
+      format_date_string(collection.updated_at)
+    end
+
+    sig { returns(T.nilable(String)) }
+    def purl
+      return if collection.druid.nil?
+
+      "https://purl.stanford.edu/#{collection.druid.gsub(/druid:/, '')}"
+    end
+
+    private
+
+    def format_date_string(date)
+      date.strftime('%B %e, %Y %I:%M%p')
```

Comment:
This should use `I18n.l(date, format: :abbr_month)`

Reply:
Thanks! One question I had at the end of the day was going to be about locale for this and the terms of use. Thanks :-)